doc/Overview/DependencyInjection/DependencyInjectionOverview.md

```diff
@@ -2,10 +2,108 @@
 uid: Overview.DependencyInjection
 ---
 
-This overview is currently under construction. It will be available soon.
+# Dependency Injection
 
-## Work in progress 🚧
+Apps built using hosting can leverage dependency injection (DI) to register services and make them available to app dependencies. This pattern enables apps to follow sound design principles, such as [SOLID](https://en.wikipedia.org/wiki/SOLID) and [Inversion of Control](https://en.wikipedia.org/wiki/Inversion_of_control). While the host builder implements standard functionality from the [Microsoft.Extensions.DependencyInjection](https://www.nuget.org/packages/Microsoft.Extensions.DependencyInjection) library, it also provides additional features to improve the developer experience.
 
-### Have questions or feedback?
+## Registering Services
 
-* Help us shape the documentation for this topic by providing feedback on the Uno.Extensions [repo](https://github.com/unoplatform/uno.extensions/discussions/categories/general)
\ No newline at end of file
+Services are registered with the host using the `ConfigureServices` method on the `IHostBuilder`. The following snippet shows how to register a service with the host:
+
+```csharp
+private IHost Host { get; }
+
+protected override void OnLaunched(LaunchActivatedEventArgs e)
+{
+    var appBuilder = this.CreateBuilder(args)
+        .ConfigureServices(services => {
```

Comment:
Update this to use the new appbuilder patter:
![image](https://user-images.githubusercontent.com/1614057/229399200-043208d2-84a9-47ad-b4d0-248279636c05.png)